federatedscope/autotune/algos.py

```diff
@@ -29,10 +28,10 @@ def make_trial(trial_cfg, client_cfgs=None):
                             server_class=get_server_cls(trial_cfg),
                             client_class=get_client_cls(trial_cfg),
                             config=trial_cfg.clone(),
-                            client_config=client_cfgs)
+                            client_configs=client_cfgs)
     results = Fed_runner.run()
     key1, key2 = trial_cfg.hpo.metric.split('.')
-    return results[key1][key2]
+    return results[key1][key2], results
```

Comment:
please supplement docstring for this function since we should let users know these changes in returned value(s).

Reply:
BTW, this makes the first returned value redundant...